### ISSUE TITLE
Add possibility to use custom channel

### ISSUE THREAD

**Issue by Zasco** [noah-severyn/SC4PackMan#41]
To fetch packages and assets metadata from a custom source other than the default channel.

**Comment by Zasco** [comment]
Partly implemented (_Zasco_'s channel only) in #48.

**Comment by noah-severyn** [comment]
@Zasco has this issue effectively been completed with the addition of the dropdowns to import YAML from various Github repos? Or did you mean something else with this?

**Comment by Zasco** [comment]
> [@Zasco](https://github.com/Zasco) has this issue effectively been completed with the addition of the dropdowns to import YAML from various Github repos? Or did you mean something else with this?

No. I meant by this to be able to define custom channels in a config and have their groups/packages/assets listed in the related field.

**Comment by noah-severyn** [comment]
So a config where a user could list any number of channels, and the 'Load from ...' and the package/asset/group dropdowns would fill from every channel in that list?

**Comment by Zasco** [comment]
> So a config where a user could list any number of channels, and the 'Load from ...' and the package/asset/group dropdowns would fill from every channel in that list?

Exactly!
